src/framework/audio/engine/internal/engineplayback.cpp

```
@@ -88,9 +88,11 @@ TrackSequenceId EnginePlayback::addSequence()
 {
     ONLY_AUDIO_ENGINE_THREAD;
 
-    static TrackSequenceId lastId = 0;
+    IF_ASSERT_FAILED(m_sequences.empty()) {
```

Cppcheck (2.20.0)

[style] The function 'isValid' is never used.

(unusedFunction)